### DISCUSSION TITLE
Theme not loading even after copy-pasting from `configuration` page

### DISCUSSION THREAD

**Discussion by IanQS** [LazyVim/LazyVim#1318] [General]
Hello! 

So I'm following the instructions: [configuration -- colorscheme](https://www.lazyvim.org/configuration#icons--colorscheme) specifically where it says 

```
Icons & colorscheme can be configured as options for the LazyVim plugin.

For example in lua/plugins/core.lua
```

and I've placed those settings into  `lua/plugins/core.lua`

```
return {
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "catppuccin",
    }
  }
}
```

---

I've tried **not** including that snippet and it seems like the colorscheme is still the same, which makes me think that it's just not actually loading.

**Comment by dpetka2001** [comment]
Did you install catppuccin? That snippet you referred to only sets up the colorscheme to be catppuccin, but it doesn't actually install catppuccin. You would need something like
```lua
  {
    "catppuccin/nvim",
    name = "catppuccin",
  },
```
to actually install catppuccin and then use the snippet you mentioned to make Lazyvim use it as default colorscheme.

**Comment by jyuan0** [comment]
Catppuccin should already be installed by default: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/colorscheme.lua#L12

----

> I've tried not including that snippet and it seems like the colorscheme is still the same, which makes me think that it's just not actually loading.

Are you sure the colorscheme is still the same? - catppuccin and tokyonight can look pretty similar. You can try changing the "flavour" of catppuccin to be sure:
```
return {
  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = "catppuccin",
    },
  },
  {
    "catppuccin/nvim",
    name = "catppuccin",
    opts = {
      flavour = "latte",
    },
  },
}
```
This should change catppuccin to a light/white background.

**Reply by dpetka2001** [reply]
@jyuan0 You are right. I was under the impression it wasn't, cuz i looked at my files and saw I had it configured there as well. My bad.

**Reply by jyuan0** [reply]
@dpetka2001 No worries! Thanks for all the questions you answer and PRs you review!

**Reply by IanQS** [reply]
Hmm, adding that snippet seemed to make things work :( I'm really not sure what's up with this..... Perhaps adding the `opts` changed things? But anyways, this helped

**Reply by jyuan0** [reply]
> Hmm, adding that snippet seemed to make things work :( I'm really not sure what's up with this..... Perhaps adding the opts changed things? But anyways, this helped

That's strange. What you originally had *should* have worked. You might want to go back to what you originally had to be sure there isn't a bug somewhere (in your local setup, or LazyVim itself). Either way, glad it helped.

**Reply by IanQS** [reply]
I might just follow the instructions and uninstall everything before doing a reinstall

**Reply by SidhantSharma398** [reply]
Thanks for the help jyuan0, I kept trying weird solutions, but this one eventually worked out. I settled on mocha and have been happy with it since!
